student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(classroom, id, age)
```

Comment:
- [ ] The `Student constructor`  extends parent's constructor by adding @classroom and a parameter for it, checkout this [link](`https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars`) for more clarification on that. Something like so;
```ruby
  def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```

Reply:
Oh nice, thank you for your honest review @Mihndim2020